src/Compilers/CSharp/Portable/Compilation/CSharpSemanticModel.cs

```diff
@@ -511,35 +531,35 @@ public SymbolInfo GetSymbolInfo(ExpressionSyntax expression, CancellationToken c
         /// </summary>
         public SymbolInfo GetCollectionInitializerSymbolInfo(ExpressionSyntax expression, CancellationToken cancellationToken = default(CancellationToken))
         {
-            CheckSyntaxNode(expression);
+                CheckSyntaxNode(expression);
```

Comment:
There are lots of indenting changes. If these are unintentional, please revert so the PR is a little easier to navigate.